### PR TITLE
'nav panel'

### PR DESCRIPTION
**Pull request by artempenteskul**

### CODE REVIEW

**Review by SerhiiPenteliuk — APPROVED** [review]

File: library/catalog/templates/collections.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{% extends 'base.html' %}
+{% load static %}
+
+{% block title %}Collections{% endblock %}
+
+{% block body %}
+
+<div class="container" style="min-height: 660px">
+
+    <h2 style="margin-top: 25px; margin-bottom: 20px;">Books with over 1000 pages</h2>
```
Comment (ivan-dubei) [inline library/catalog/templates/collections.html]: Move inline CSS to the separate file.

File: library/catalog/templates/collections.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    <div class="swiper-container first-container">
+      <!-- Additional required wrapper -->
+      <div class="swiper-wrapper">
+        <!-- Slides -->
+          {% for book in pages_books %}
+            <div class="swiper-slide" style="position: relative; margin-bottom: 10px;">
```
Comment (ivan-dubei) [inline library/catalog/templates/collections.html]: Move inline CSS to the separate file.

File: library/catalog/templates/collections.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+                <a href="{% url 'book-details' book.id %}">
+                    <img src="{{ book.link_img }}" alt="{{ book.title }}">
+                    <p><b>{{ book.title }}</b></p>
+                    <p class="text-bottom">Pages: {{ book.pages }}</p>
+                </a>
+            </div>
+          {% endfor %}
+      </div>
+    </div>
+    <div class="swiper-button-next first-next" style="position: relative; top: -278px; left: 1150px"></div>
```
Comment (ivan-dubei) [inline library/catalog/templates/collections.html]: Move inline CSS to the separate file.

File: library/catalog/templates/news_page.html [hunk 4]
```diff
@@ -0,0 +1,13 @@
+{% extends 'base.html' %}
+{% load static %}
+
+{% block title %}News{% endblock %}
+
+{% block body %}
+
+<div class="container" style="min-height: 650px">
```
Comment (ivan-dubei) [inline library/catalog/templates/news_page.html]: Move inline CSS to the separate file.

File: library/catalog/templates/collections.html (hunk 1 above)
Comment (artempenteskul) [reply library/catalog/templates/collections.html]: done

File: library/catalog/templates/collections.html (hunk 2 above)
Comment (artempenteskul) [reply library/catalog/templates/collections.html]: done

File: library/catalog/templates/collections.html (hunk 3 above)
Comment (artempenteskul) [reply library/catalog/templates/collections.html]: I moved CSS to a separate file, but I used !important statement due to priority

File: library/catalog/templates/news_page.html (hunk 4 above)
Comment (artempenteskul) [reply library/catalog/templates/news_page.html]: done

**Review by ivan-dubei — APPROVED** [review]

**Pull request merged**